### ISSUE TITLE
[feature] Anonymous Dashboard View

### ISSUE THREAD

**Issue by captn3m0** [go-gitea/gitea#3263]
- Gitea version (or commit ref): 1.3.2
- Git version: 
- Operating system: Arch Linux
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist:

## Description

Right now the Gitea homepage for anonymous users is a landing page. It might make more sense to show a public dashboard view (A Dashboard view for a user with only public read access)

I'm primarily running my gitea server for my own code (and not really as a service), so it makes sense to showcase current activity, instead of a landing page.

Along with https://github.com/go-gitea/gitea/issues/1745, it could be set up as a admin setting:

Pick your Home Page:

- [ ] Landing Page
- [ ] Pick an Organization
- [ ] Site Dashboard
- [ ] Site-wide Explore

## Screenshots
Basically for someone who opens `git.captnemo.in`, they currently see see:

![image](https://user-images.githubusercontent.com/584253/34380636-fee3eeea-eb28-11e7-9ceb-a9ca0b3757c8.png)

Instead I want them to see (bad mockup):

![image](https://user-images.githubusercontent.com/584253/34380678-598585ac-eb29-11e7-9617-cdb02a03287b.png)

**Comment by captn3m0** [comment]
I would be interested in working on this, if others thing this might be useful.

**Comment by lafriks** [comment]
At first home needs to be made so that it can display all public activity as currently it displays info related to currently logged in user

**Comment by captn3m0** [comment]
I removed the right-side repositories listing because that doesn't make much sense in anonymous view. But yeah, the template might need some changes for this to work.

**Comment by ethantkoenig** [comment]
Conversely, there are situations where it would be nice for logged-in users to be able to see the landing page (not necessarily as the default, but right now they can't see it at all).

**Comment by captn3m0** [comment]
Does a `/home` route make sense? That always serves the landing page?

Just checked after writing this that github.com/home also serves you the GitHub landing page, although without the signup form.

**Comment by lafriks** [comment]
It could be either home or it's subroute

**Comment by captn3m0** [comment]
While this is not completely resolved, anonymous users reach the explore repos page on my setup by default (git.captnemo.in), which is far better.
